test/functional/glance_test_data.go

```diff
@@ -43,7 +43,7 @@ const (
 // GlanceTestData is the data structure used to provide input data to envTest
 type GlanceTestData struct {
 	ContainerImage              string
-	GlanceDatabaseUser          string
+	GlanceDatabaseAccount       string
```

Comment:
See my previous comment: if this is turned into a `types.NamespacedName` and defined with:

```golang
GlanceDatabaseAccount : types.NamespacedName{
	Namespace: glanceName.Namespace,
	Name:      GlanceServiceUser,
},
```
we can reduce the code and we still have the chance to compare the expected name by accessing `glanceTest.GlanceDatabaseAccount.Name`.